### PR TITLE
New Exercise: isogram

### PR DESCRIPTION
**Pull request by jvarness**
I tried thinking of a phrase to put here that's an isogram but it's actually really hard.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -216,6 +216,16 @@
         "pattern_matching",
         "stacks"
       ]
+    },
+    {
+      "slug": "isogram",
```
Comment (Stargator) [inline config.json]: Recommend moving this to after armstrong-numbers

File: config.json [hunk 2]
```diff
@@ -216,6 +216,16 @@
...
+      "uuid": "3c1b70fa-aaa8-4323-9aab-4c76da0a89e1",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "strings"
```
Comment (Stargator) [inline config.json]: You could add logic and/or algorithms as topics

File: exercises/isogram/test/isogram_test.dart [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import 'package:test/test.dart';
+import 'package:isogram/isogram.dart';
```
Comment (Stargator) [inline exercises/isogram/test/isogram_test.dart]: Please sort the imports alphabetically.

File: exercises/isogram/test/isogram_test.dart [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+void main() {
+  final isogram = new Isogram();
+
+  group('Isogram', () {
+    group('Check if the given string is an isogram', () {
+      test("empty string", () {
+        final bool result = isogram.isIsogram("");
+        expect(result, equals(true));
+      }, skip: false);
```
Comment (Stargator) [inline exercises/isogram/test/isogram_test.dart]: All other tests after this should be `skip: true`

File: exercises/isogram/test/isogram_test.dart (hunk 3 above)
Comment (jvarness) [reply exercises/isogram/test/isogram_test.dart]: Done!

File: exercises/isogram/test/isogram_test.dart (hunk 4 above)
Comment (jvarness) [reply exercises/isogram/test/isogram_test.dart]: Skipped all but the first.

File: config.json (hunk 1 above)
Comment (jvarness) [reply config.json]: Moved.

File: config.json (hunk 2 above)
Comment (jvarness) [reply config.json]: Added both :)

**Comment by jvarness** [comment]
@Stargator @devkabiir addressed comments. We good on this one?

**Comment by Stargator** [comment]
@jvarness, I'll take a look when I get home. Thanks for the heads up

File: config.json [hunk 5]
```diff
@@ -57,6 +57,18 @@
         "strings"
       ]
     },
+    {
+      "slug": "isogram",
+      "uuid": "3c1b70fa-aaa8-4323-9aab-4c76da0a89e1",
+      "core": false,
+      "unlocked_by": null,
```
Comment (Stargator) [inline config.json]: @jvarness unlocked_by should have "armstrong-numbers" as the value, unless you want to make it a core exercise.

File: config.json [hunk 6]
```diff
@@ -57,6 +57,18 @@
...
+      "difficulty": 1,
+      "topics": [
+        "strings",
+        "algorithms",
+        "logic"
```
Comment (Stargator) [inline config.json]: I think configlet will want the topics to be sorted alphabetically

File: config.json (hunk 5 above)
Comment (jvarness) [reply config.json]: Oh I wasn't sure if you meant the position in the file or if you wanted it to be unlocked by armstrong numbers. I'll change that.

File: config.json (hunk 6 above)
Comment (jvarness) [reply config.json]: Alphabetized

File: config.json (hunk 5 above)
Comment (jvarness) [reply config.json]: Set `"unlocked_by": "armstrong-numbers"`

**Pull request merged**